cmd/engine/engine_test.go

```diff
@@ -9,6 +11,11 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func init() {
```

Comment:
Yes. Test_Main is probably the better way to initialise it.

Reply:
hmm although this would be a major drawback, that we would need a lot of TestMains for test files where we have none. The only issue that remains would be if this somehow overrides a log that exists in the same package already?

Reply:
I deleted all init() methods from the test files.